### PR TITLE
Books Show

### PR DESCRIPTION
**Pull request by nvatuan**
## Related Tickets
- [#46936](https://edu-redmine.sun-asterisk.vn/issues/46936)

## WHAT (optional)
- Added screen for `Book`'s `show` action
- Wrote code to populate `Book` in `db/seeds.rb`

## HOW
- Added HTML + bootstrap and some css to display `@book` attributes

## Evidence (Screenshot or Video)
Books Show

Framgia-ci

### CODE REVIEW

**Comment by txphuc** [comment]
Ok

**Comment by nvatuan** [comment]
ready

**Comment by nvatuan** [comment]
ready

File: app/models/category.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 class Category < ApplicationRecord
   has_many :books, dependent: :nullify
+
+  belongs_to :parent, class_name: "Category", foreign_key: "parent_id",
```
Comment (ghost) [inline app/models/category.rb]:
```suggestion
  belongs_to :parent, class_name: Category.name, foreign_key: :parent_id,
```
Sửa tương tự như ở bên dưới

File: app/models/category.rb [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+              optional: true
+  has_many :children, class_name: "Category", foreign_key: "parent_id",
+            dependent: :nullify
```
Comment (ghost) [inline app/models/category.rb]: Sao ở đây lại `dependent: :nullify` e

File: app/views/books/show.html.erb [hunk 3]
```diff
@@ -1,16 +1,43 @@
...
-    </section>
-  </aside>
+<div class="book-details">
+  <h1><%= t ".book" %></h1>
+  <hr>
+
+  <div class="row book-info">
+    <h2 class="book-title "><%= @book.title %></h2>
+    <div class="col-xs-3">
+      <%= image_tag get_book_cover(@book), alt: t(".default_book_alt"), class: "book-cover"  %>
```
Comment (ghost) [inline app/views/books/show.html.erb]:
```suggestion
      <%= image_tag get_book_cover(@book), alt: t(".default_book_alt"), class: "book-cover" %>
```

**Review by ghost — COMMENTED** [review]
cmt

File: app/models/category.rb (hunk 1 above)
Comment (nvatuan) [reply app/models/category.rb]: Fixed

File: app/models/category.rb (hunk 2 above)
Comment (nvatuan) [reply app/models/category.rb]: Dạ để em sửa lại thành `dependent: :destroy`

Comment (nvatuan) [reply app/models/category.rb]: Fixed

File: app/views/books/show.html.erb (hunk 3 above)
Comment (nvatuan) [reply app/views/books/show.html.erb]: Fixed

**Comment by nvatuan** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**